store/sqlstore/post_store.go

```diff
@@ -1202,3 +1230,66 @@ func (s SqlPostStore) GetOldest() store.StoreChannel {
 		result.Data = &post
 	})
 }
+
+func (s *SqlPostStore) determineMaxPostSize() int {
+	var maxPostSize int = model.POST_MESSAGE_MAX_RUNES_V1
+	var maxPostSizeBytes int32
+
+	if s.DriverName() == model.DATABASE_DRIVER_POSTGRES {
+		// The Post.Message column in Postgres has historically been VARCHAR(4000), but
+		// may be manually enlarged to support longer posts.
+		if err := s.GetReplica().SelectOne(&maxPostSizeBytes, `
+			SELECT
+				COALESCE(character_maximum_length, 0)
+			FROM
+				information_schema.columns
+			WHERE
+				table_name = 'posts'
+			AND	column_name = 'message'
+		`); err != nil {
+			l4g.Error(utils.T("store.sql_post.query_max_post_size.error") + err.Error())
```

Comment:
This should be in the "architecture decisions" document once it's made, but we decided a while back that we don't need to translate server errors that are just logged to the console any more, just ones that are sent back through the API. You don't need to change this back, but it's something that'll save some time in the future